src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -45,6 +47,35 @@ var eventTypes = {
   },
 };
 
+function getInstIfValueChanged(targetInst, target) {
+  var value = getInputValue(target);
+  var lastValue = target[LAST_VALUE_KEY];
+
+  if (!target.hasOwnProperty(LAST_VALUE_KEY) || value !== lastValue) {
```

Comment:
Not sure that tracking on the DOM node is the best option, perhaps the instance is better? I am not sure with object is more long-lived